server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/user/User.java

```diff
@@ -0,0 +1,72 @@
+package de.tum.in.www1.hephaestus.codereview.user;
+
+import jakarta.persistence.Table;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.springframework.lang.NonNull;
+
+import de.tum.in.www1.hephaestus.codereview.base.BaseGitServiceEntity;
+import de.tum.in.www1.hephaestus.codereview.comment.IssueComment;
+import de.tum.in.www1.hephaestus.codereview.pullrequest.PullRequest;
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.OneToMany;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Entity
+@Table(name = "user", schema = "public")
```

Comment:
What is `schema = "public"` can we remove it?

Reply:
The table name `user` is a reserved keyword in Postgres. That's why we have to specify the schema, see the [PostgreSQL Docs](https://www.postgresql.org/docs/8.1/sql-keywords-appendix.html) or this [Stackoverflow post](https://stackoverflow.com/questions/45782327/org-postgresql-util-psqlexception-error-column-user0-id-does-not-exist-hibe)